tools/depth-quality/CMakeLists.txt

```diff
@@ -6,8 +6,10 @@ project( rs-depth-quality )
 
 if(WIN32)
     add_executable( ${PROJECT_NAME} WIN32 "" )
+    list( APPEND DEPENDENCIES OpenGL::GL)
```

Comment:
Why not add a dependency to the app only?
This add also a dependency for the SDK right?

Reply:
I am not sure about it :)

Reply:
changed, now we add it using target_link_libraries() instead of appending it to the DEPENDENCIES  list.